package.json

```diff
@@ -27,7 +25,8 @@
     "angular-docs",
     "environments",
     "examples/*",
-    "packages/*"
+    "packages/*",
+    "packages/angular/projects/ui-angular"
```

Comment:
This subdirectory, not `packages/angular`, contains the actual library code we want to publish. `packages/angular` is more of Angular workspace root to provide infrastructure for Angular development. Relevant docs: [link](https://angular.io/guide/creating-libraries#getting-started).

Reply:
What's the primary difference between how Angular works in this repo vs. https://github.com/aws-amplify/amplify-js/tree/main/packages/aws-amplify-angular?

Reply:
1) We're now using [`ng-packagr`](https://github.com/ng-packagr/ng-packagr) + `@angular/cli` to build packages. This outputs more bundles than we used to according to [Angular Package Format](https://github.com/ng-packagr/ng-packagr).

2) We used to build our packages somewhat [manually](https://github.com/aws-amplify/amplify-js/blob/main/packages/aws-amplify-angular/package.json#L10-L14), with explicit configurations. Now these are being done automatically with `ng build`.

3) On our legacy component, we declared most of Angular dependency as [`devDependency`](https://github.com/aws-amplify/amplify-js/blob/main/packages/aws-amplify-angular/package.json#L29-L68). Angular package in this repo does this by a mini workspace structure -- these angular infrastructure are defined in `packages/angular` root `package.json`.  The actual library code is defined in `packages/angular/projects/ui-angular` subdirectory. Its [`package.json`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/angular/projects/ui-angular/package.json#L5-L6) is free of these angular infrastructure because they are hoisted are at its parent.

Reply:
To visualize this, 
```bash
packages/angular/
├── projects/ui-angular/
│   ├── src/ # contains src files for actual library code.
│   └── package.json # name: `@aws-amplify/ui-angular`, private: false
│                        # contains dependencies that library needs (e.g. xstate)
│
└── package.json # name: 'amplify-ui-angular-mono`, private: true
                 # contains base dependencies for developing angular packages